examples/WebApp/WebApp.Tests.ps1

```diff
@@ -119,8 +119,8 @@ Describe 'Verify Web App Static Site Exists' {
     # The '-ErrorAction SilentlyContinue' command suppresses all errors.
     # In this test, it will suppress the error message when a resource cannot be found.
     # Remove this field to see all errors.
-    Confirm-AzBPWebAppStaticSite -ResourceGroupName $rgName -StaticWebAppName $nowebappStaticSiteName -ErrorAction SilentlyContinue
-    | Should -Not -BeSuccessful
+    Confirm-AzBPWebAppStaticSite -ResourceGroupName $rgName -StaticWebAppName $nowebappStaticSiteName `
```

Comment:
Do not use backticks. Instead use splatting.

```suggestion
    # arrange
    # The '-ErrorAction SilentlyContinue' command suppresses all errors.
    # In this test, it will suppress the error message when a resource cannot be found.
    # Remove this field to see all errors.
    $params = @{
      ResourceGroupName = $rgName
      StaticWebAppName  = $nowebappstaticsitename
      ErrorAction       = "SilentlyContinue"
    }
    
    # act and assert
    Confirm-AzBPWebAppStaticSite @params | Should -Not -BeSuccessful
```